### PR TITLE
log volumesnapshotbackup status if is not complete.

### PR DESCRIPTION
**Pull request by kaovilai**
Thank you for contributing to Velero!

# Please add a summary of your change

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [x] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Review by eemcmullan — DISMISSED** [review]

File: pkg/datamover/datamover.go [hunk 1]
```diff
@@ -67,13 +68,13 @@ func CheckIfVolumeSnapshotBackupsAreComplete(ctx context.Context, volumesnapshot
...
-				err := volumeSnapMoverClient.Get(ctx, kbclient.ObjectKey{Namespace: volumesnapshotbackup.Namespace, Name: volumesnapshotbackup.Name}, &tmpVSB)
+				currentVSB := snapmoverv1alpha1.VolumeSnapshotBackup{}
+				err := volumeSnapMoverClient.Get(ctx, kbclient.ObjectKey{Namespace: volumesnapshotbackup.Namespace, Name: volumesnapshotbackup.Name}, &currentVSB)
 				if err != nil {
 					return false, errors.Wrapf(err, fmt.Sprintf("failed to get volumesnapshotbackup %s/%s", volumesnapshotbackup.Namespace, volumesnapshotbackup.Name))
 				}
-				if len(tmpVSB.Status.Phase) == 0 || tmpVSB.Status.Phase != snapmoverv1alpha1.SnapMoverBackupPhaseCompleted {
-					log.Infof("Waiting for volumesnapshotbackup to complete %s/%s. Retrying in %ds", volumesnapshotbackup.Namespace, volumesnapshotbackup.Name, interval/time.Second)
+				if len(currentVSB.Status.Phase) == 0 || currentVSB.Status.Phase != snapmoverv1alpha1.SnapMoverBackupPhaseCompleted {
+					log.Infof("Waiting for volumesnapshotbackup status.phase to change from %s to complete %s/%s. Retrying in %ds", currentVSB.Status.Phase, volumesnapshotbackup.Namespace, volumesnapshotbackup.Name, interval/time.Second)
```
Comment (eemcmullan) [inline pkg/datamover/datamover.go:77]: Would it be possible to hit nil pointer ref here if `currentVSB.Status.Phase` doesn't exist yet? It is only included once a replicationSource is created.

Comment (kaovilai) [reply pkg/datamover/datamover.go:77]: It is not possible to hit nil pointer here.
`VolumeSnapshotBackupStatus` from this PR's go.mod indicates that `status.Phase` is not a pointer and has an underlying type of string. Therefore, uninitialized, status.Phase should be `""`. And if somehow it was nil, you can certainly `Printf("%s", nil)`

```
type VolumeSnapshotBackupStatus struct {
	Completed bool `json:"completed,omitempty"`
	// Include references to the volsync CRs and their state as they are
	// running
	Conditions []metav1.Condition `json:"conditions,omitempty"`
	// Includes source PVC name and size
	SourcePVCData PVCData `json:"sourcePVCData,omitempty"`
	// Includes restic repository path
	ResticRepository string `json:"resticrepository,omitempty"`
	// volumesnapshot backup phase status
	Phase VolumeSnapshotBackupPhase `json:"phase,omitempty"`
	// name of the VolumeSnapshotClass
	VolumeSnapshotClassName string `json:"volumeSnapshotClassName,omitempty"`
}

type VolumeSnapshotBackupPhase string
```

Comment (eemcmullan) [reply pkg/datamover/datamover.go:77]: cool

**Review by eemcmullan — APPROVED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**